backend/src/routes/Call.ts

```diff
@@ -1,28 +1,89 @@
 import express, { Router } from 'express';
 import * as z from 'zod';
+import _ from 'lodash';
 import { requireSelf } from './middlewares';
 import type {
   Call as CallService,
   TwilioCall as TwilioCallService,
+  Contact as ContactService,
 } from '../services';
 import Stopwatch from '../util/stopwatch';
 import {
   validateRequest,
   stringToNumberTransformer,
 } from './helpers/validation';
+import { Call, Contact } from '../models';
+import TwilioCall from '../models/TwilioCall';
 
 const GET_SCHEMA = z.object({
   params: z.object({
     userId: stringToNumberTransformer,
   }),
 });
 
+const GET_RECENT_SCHEMA = z.object({
+  params: z.object({
+    userId: stringToNumberTransformer,
+  }),
+});
+
+function mapObjectsToRecentCallResponse(
```

Comment:
wanted to use sequelize includes in query, but I couldn't get the types to work. Think we can use this first and change it next time.